nodejs/week3/index.js

```diff
@@ -0,0 +1,85 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+//warmup
+//calculator
+//Getting data through query parameters using GET
+//multiply
+app.get("/calculator/multiply", (req, res) => {
```

Comment:
Bonus:
This code works but it's not easy to read and maintain.
You can imagine how this kind of code changes if I now say that we now can have 3 operands.
You'll need a lot of `if` statements to support that. 

Instead, you should aim to make your code converge on a shared format! 
If you could assume that your code always deals with lists, it would be a lot simpler.

So split it into two steps:
1. Turn parameters into lists always
2. Use lists to perform operations

See [this ](https://github.com/negaren/hyf-homework/pull/19/files#r826736613) comment for more advice IF you get stuck.